src/SingleContainer/Orchestrator/NetworkPerspective.Sync.Orchestrator.Tests/ComponentTests/ConnectorsControllerTests.cs

```diff
@@ -34,13 +34,13 @@ public async Task ShouldReturnCreatedConnectors()
         await wokrersClient.WorkersPostAsync(new CreateWorkerDto { Name = workerName, Secret = "secret1" });
 
         var workers = await wokrersClient.WorkersGetAsync();
-        var workerId = workers.Single(x => x.Name == workerName).Id;
+        var workerId = workers.Single(x => x.Name == workerName).Id as Guid?;
```

Comment:
Is this change needed?

Reply:
Tak. Ale chyba bardziej problemem jest klient c# wygenerowany przez nswag, który nazywa metody wg http verbs a nie korzysta z nazw metod w kontrolerze (co jest domyślnym ustawieniem w Nswag).

Dodałem w kontrolerze drugą metodę GET i w kliencie są dwie takie same metody (o tej samej nazwie) - jedna przyjmuje guid a druga guid?. Powinny się inaczej nazywać ale to wymaga zmiany w konfiguracji nswag i pewnie wszystkich testów więc tego już nie ruszałem.